README.md

```diff
@@ -11,18 +11,22 @@ Intended for [gettext](https://www.npmjs.com/search?q=gettext) or similar transl
 This library provides custom [Tagged Template Literals](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals#Tagged_template_literals) interpolation functions.
 
 ```javascript
-tag = (template, ...substitutions) => ...
+const tag = (template, ...substitutions) => ...
+const substitution = 10;
 
 tag`Some statement that can use ${substitution}`
+// => Some statement that can use 10
 ```
 
-Template Literals allow **substitutions**. These will appear as wildcards in the untranslated text.
+Template Literals allow **substitutions**. These will appear as **tokens** in the untranslated text.
 
 ```
 msgid "Some statement that can use ____"
```

Comment:
So you want to keep non-object replacements?
I reckon you then use number there from `i` which would resolve to `__0__`